airflow/api_connexion/endpoints/role_and_permission_endpoint.py

```diff
@@ -147,7 +147,7 @@ def post_role() -> APIResponse:
     if not role:
         perms = [(item['action']['name'], item['resource']['name']) for item in data['permissions'] if item]
         _check_action_and_resource(security_manager, perms)
-        security_manager.init_role(role_name=data['name'], perms=perms)
+        security_manager.bulk_sync_roles([{"role": data["name"], "perms": perms}])
```

Comment:
Seems unrelated?

Reply:
`init_role()` is deprecated (and simply calls `bulk_sync_roles`), this removes one deprecation warning.